### DISCUSSION TITLE
"wt.exe: The command parameter was already specified" with split-pane and pwsh.exe

### DISCUSSION THREAD

**Discussion by tesar-tech** [microsoft/terminal#12685] [Q&A]
I am trying to open multiple split-panes. Both would load PowerShell profile and both would run pwsh command just from the start..
I came up with this:

```ps1
wt -d "C:\path\to\somewhere" -p "PowerShell" pwsh.exe -Command {dotnet --version} --noExit `; split-pane -d "C:\path\to\somewhere\else" -p "PowerShell"  pwsh.exe -Command {dotnet --version} --noExit
``` 
Which should print dotnet version to both panes. But it results in:

> wt.exe: The command parameter was already specified.

When I remove the command from second pane, it runs just fine (but doesn't print dotnet version obviously)

```ps1
wt -d "C:\path\to\somewhere" -p "PowerShell" pwsh.exe -Command {dotnet --version} --noExit `; split-pane -d "C:\path\to\somewhere\else" -p "PowerShell"  pwsh.exe --noExit
```
What is wrong here?

(In real-world scenario I am using this to run both Blazor WA client and hosting asp.net core api)

**Comment by DannyMeister** [comment]
It is the curly braces which cause this problem when I try to run multiple pwsh commands.  I have been able to get my commands to work by making `-Command` the final parameter to pwsh.exe and omitting the curly braces.  For example, this works fine for me:

``` powershell
wt -p "PowerShell" pwsh.exe --noExit -Command dotnet --version `; split-pane -p "PowerShell"  pwsh.exe --noExit -Command dotnet --version
```
